test/with_dummyserver/test_proxy_poolmanager.py

```diff
@@ -67,21 +77,31 @@ def teardown_class(cls) -> None:
         super().teardown_class()
         shutil.rmtree(cls.certs_dir)
 
-    def test_basic_proxy(self) -> None:
+    def test_proxy_verified_http_http(self) -> None:
         with proxy_from_url(self.proxy_url, ca_certs=DEFAULT_CA) as http:
             r = http.request("GET", f"{self.http_url}/")
             assert r.status == 200
+            assert_is_verified(http, False, False)
 
+    def test_proxy_verified_http_https(self) -> None:
+        with proxy_from_url(self.proxy_url, ca_certs=DEFAULT_CA) as http:
             r = http.request("GET", f"{self.https_url}/")
             assert r.status == 200
+            assert_is_verified(http, False, True)
 
-    def test_https_proxy(self) -> None:
+    def test_proxy_verified_https_http(self) -> None:
         with proxy_from_url(self.https_proxy_url, ca_certs=DEFAULT_CA) as https:
-            r = https.request("GET", f"{self.https_url}/")
+            r = https.request("GET", f"{self.http_url}/")
             assert r.status == 200
+            # TODO this possibly should be "True, False"
+            #      see https://github.com/urllib3/urllib3/issues/3267
+            assert_is_verified(https, True, True)
```

Comment:
I am reluctant to adding a passing test that ensures that `is_verified` is set to a wrong value

Reply:
I understand that you are sure about the value.

I can change to 'True' and skip the failing test.

Reply:
=> ` assert_is_verified(https, True, False)`
=> skip test